src/web/mjs/connectors/Copymanga.mjs

```diff
@@ -1,16 +1,117 @@
 import Connector from '../engine/Connector.mjs';
 import Manga from '../engine/Manga.mjs';
 
+function formatDate() {
+    // Example: 2023.03.27
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = now.getMonth() + 1;
+    const day = now.getDate();
+    return `${year}.${month < 10 ? "0" : ""}${month}.${day}`;
```

Comment:
The `day` may be a single digit, is this by intention?

Reply:
No it's not intended, my mistake.